keyvi/tests/keyvi/dictionary/fsa/internal/memory_map_manager_test.cpp

```diff
@@ -23,33 +23,31 @@
  *      Author: hendrik
  */
 
-#include <cstring>
-#include <boost/test/unit_test.hpp>
-
 #include "dictionary/fsa/internal/memory_map_manager.h"
+#include <boost/test/unit_test.hpp>
+#include <cstring>
```

Comment:
clang-format does not get it right, the order should be:

- C includes
- empty line
- C++ includes
- empty line
- boost includes
- empty line
- other 3rdparty includes
- empty line
- keyvi includes except for trace
- empty line
- trace include, with comment for enabling it

Reply:
Thanks Hendrik for the information. Is it possible that clang-format behaves differently on different platform? I see one of the builds is failing if I correct the order. How should I proceed here if the clang-format doesn't accept the aforementioned order.